File: 3843/3843.py
class Solution:
    def first_element_with_unique_frequency(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.first_element_with_unique_frequency(root.left), self.first_element_with_unique_frequency(root.right))

File: 3843/3843.go
func firstElementWithUniqueFrequenc(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := firstElementWithUniqueFrequenc(root.Left)
	r := firstElementWithUniqueFrequenc(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}